ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-cron/gio-form-cron.adapter.ts

```diff
@@ -184,8 +188,8 @@ export const toCronDescription = (cronExpression: string): string => {
 const cronIsValid = (cronExpression: string): boolean => {
   const parts = cronExpression.split(' ');
 
-  if (parts.length < 5 || parts.length > 6) {
-    return false;
+  if (parts.length != 6) {
+    throw new Error('Cron expression must have 6 parts.');
```

Comment:
Issue: maybe you need to trim the cron expression before you split. 
If I enter "* * * * * " (notice the trailing space) -> I have no error
If I enter "* * * * *" (no space at the end) -> I have an error.

Reply:
I could make these 2 scenarios work.
But I wonder if it shouldn't be up to the rest-api to validate or not. Custom mode means that you know what you're doing. WDYT ?

Reply:
it feels strange to have an error in only one of the 2 cases. I would either implement strict check with trim, or no check and let the back validate (a minimum check on the front will give quicker feedback loop so I would choose the first option).

Also, the number of parts in cron expressions can be 5, 6, or 7 depending on the systems. Are we sure that we will always need 6 parts ? Or maybe it's an input param to add to the component ?

Reply:
Yes, yes, I'm going to improve the spacing and the trim. I'm talking in general terms.

No they are cron spring they are always 6 part https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/support/CronExpression.html

Reply:
added WDYT ? I could validate more severely, but I might be limiting the custom for nothing.

Reply:
Yes I think the basic check is ok now. If we need more advanced validation, I agree it can be done on the backend side